e2etests/node_scan_rhcos_test.go

```diff
@@ -175,12 +176,12 @@ func TestGRPCGetRHCOSNodeVulnerabilities(t *testing.T) {
 
 // assertIsSubset asserts that every element of expectedToExist exists in gotVulns
 func assertIsSubset(t *testing.T, gotVulns, expectedToExist []*v1.Vulnerability) {
+	assert.GreaterOrEqual(t, len(gotVulns), len(expectedToExist), "Expected %d vulnerabilities to be a subset of a set that has %d elements", len(gotVulns), len(expectedToExist))
```

Comment:
The message is misleading as the "subset"/order of arguments are mixed.
Either it should be changed to
```go
	assert.GreaterOrEqual(t, len(gotVulns), len(expectedToExist), "Expected %d vulnerabilities to be a superset of a set that has %d elements", len(gotVulns), len(expectedToExist))
```
or just drop `%d`s and provide a text. I think the library will print `e1` and `e2` argument values.

Reply:
Improved in [6692db4](https://github.com/stackrox/scanner/pull/1004/commits/6692db429eba7737b9c090e679f5fe5dad836a39)